source/io/CsvReader.ooc

```diff
@@ -2,8 +2,6 @@ use ooc-base
 use ooc-collections
 import io/[File, FileReader]
 
-// TODO: Handle multi-line string literals
```

Comment:
Discussed in October that we have no use for this atm https://github.com/cogneco/ooc-kean/issues/566